### ISSUE TITLE
We haven't documented core dumps

### ISSUE THREAD

**Issue by palmer-dabbelt** [riscv/riscv-elf-psabi-doc#170]
QEMU is about to start generating these.

**Comment by kito-cheng** [comment]
More info about this topic:

Andrew Burgess from Embecosm has implement coredump for bare-metal RISC-V last month, @jim-wilson has suggest we should have document that.

[1] http://sourceware-org.1504.n7.nabble.com/Re-PATCH-0-8-Bare-metal-core-dumps-for-RISC-V-td655731.html

**Comment by palmer-dabbelt** [comment]
I agree that it's not a blocker to have this documented, as it's defacto stable ABI just by virtue of being all over the place (IIUC Linux core dumps have worked in GDB for a while).  I was just trying to list stuff that would be nice to do, there's a lot of them in here ;)
